.github/actions/aurora-manage-cluster/action.yml

```diff
@@ -0,0 +1,202 @@
+name: Deploy RDS Aurora Cluster
+
+description: |
+  This GitHub Action automates the deployment of an RDS Aurora cluster using Terraform.
+  This action will also install Terraform and awscli. It will output the Aurora cluster endpoint.
+
+inputs:
+  cluster-name:
+    description: 'Name of the RDS Aurora cluster to deploy'
+    required: true
+
+  engine-version:
+    description: 'Version of the Aurora engine to use'
+    required: true
+    # renovate: datasource=endoflife-date depName=amazon-rds-postgresql versioning=semver
+    default: "15.4"
+
+  instance-class:
+    description: 'Instance class for the Aurora cluster'
+    required: true
+    default: "db.t3.medium"
```

Comment:
general style suggestion for the whole document, align on single vs double quotes.
I tried finding a pattern on your usage but there wasn't any. First thought you always do description single quote and default double quote but that didn't hold up.